dist/tools/setsid/Makefile

```diff
@@ -6,7 +6,7 @@ PKG_BUILDDIR = $(CURDIR)/bin
 
 .PHONY: all
```

Comment:
Needs to be removed and `$(RIOTBASE)/pkg/pkg.mk` moved up

Reply:
> > I'm just answering regarding the u8g2 .c file. I think there's no need for copying it but it could just be placed in a contrib subfolder with a Makefile to create a single module, let's say u8g2_riot.
> > Then this module would just be a dependency of the u8g2 package (use USEMODULE += u8g2_riot in a Makefile.dep file.
> > Note that the other Makefiles in csrc and subfolders could be renamed Makefile.csrc, Makefile.sys and then, instead of copying them, you could call make -C $(PKG_BUILDDIR)//csrc -f $(RIOTBASE)/pkg/u8g2/Makefile.csrc, etc.
> > I think that would make the package cleaner and easier to maintain.
> > I can help here if you want :)
> 
> Changing to ` cp -u $(RIOTBASE)/pkg/u8g2/src/csrc/u8g2_riotos.c $(PKG_BUILDDIR)/csrc/u8g2_riotos.c` also works

Changed it to this as a work-around for now.

Reply:
Fixed in e6aaf1e.